### PR TITLE
OVN: bump to ovs-2.12

### PR DESCRIPTION
**Pull request by pecameron**
SDN-611
https://jira.coreos.com/browse/SDN-611

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@dcbw PTAL

File: Dockerfile [hunk 1]
```diff
@@ -32,11 +32,10 @@ RUN yum install -y  \
...
 RUN INSTALL_PKGS=" \
-	PyYAML bind-utils openssl firewalld-filesystem \
-	libpcap  hostname iproute strace socat \
-	openvswitch2.11 openvswitch2.11-devel \
-	openvswitch2.11-ovn-common openvswitch2.11-ovn-central \
-	openvswitch2.11-ovn-host openvswitch2.11-ovn-vtep \
+	PyYAML openssl firewalld-filesystem \
+	libpcap iproute strace \
+	openvswitch2.12 openvswitch2.12-devel \
+	ovn2.12 ovn2.12-central ovn2.12-host ovn2.12-vtep \
```
Comment (dcbw) [inline Dockerfile]: @pecameron so we don't actually want OVN 2.12; just OVS 2.12

Comment (pecameron) [reply Dockerfile]: OK.

**Pull request merged**